canonical/canonical.com
# Add new pages to /solutions/telco bubble

## Done

- New pages /solutions/telco/5g-edge and /solutions/telco/5g-core

## QA

- Go to https://canonical-com-1385.demos.haus/solutions/telco/5g-edge and https://canonical-com-1385.demos.haus/solutions/telco/5g-core
- Check that the pages are up and working as expected
- The PRs have already been checked individually, so what is required is just a final QA check

## Issue / Card

Fixes #

## Screenshots

[if relevant, include a screenshot]
